src/server/judge/settings.py

```diff
@@ -25,7 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = ['127.0.0.1', '192.168.43.101']
```

Comment:
Perhaps the commit message could be better over here, something like "settings: Allow all hosts to access."